### PR TITLE
Add osasinfra-ipv6 network-values template

### PR DESCRIPTION
**Pull request by rlobillo**
Needed for https://github.com/openstack-k8s-operators/architecture/pull/421

### CODE REVIEW

**Comment by lewisdenny** [comment]
@rlobillo Please set the PR to ready state to remove the draft status if it's ready for review and merging.

File: reproducer.yml
[the remark this answers is not in the document]
Comment (rlobillo) [reply reproducer.yml:1]: Yeah, I understand your point. However, it's kind of related, as this change is a consequence of osasinfra-ipv6 deployment...

**Pull request merged**